ReactWindows/ReactNative.Shared/ReactNative.Shared.projitems

```diff
@@ -28,12 +29,23 @@
     <Compile Include="$(MSBuildThisFileDirectory)Bridge\JavaScriptModuleBase.Generated.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Bridge\JavaScriptModuleRegistration.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Bridge\JavaScriptModuleRegistry.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Bridge\NativeArgumentsParseException.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Bridge\Queue\IMessageQueueThread.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Bridge\Queue\IReactQueueConfiguration.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Bridge\Queue\MessageQueueThreadExtensions.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Bridge\Queue\MessageQueueThreadKind.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Bridge\Queue\MessageQueueThreadSpec.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Bridge\Queue\ReactQueueConfigurationSpec.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Bridge\ReactDelegateFactoryBase.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Bridge\ReactMethodAttribute.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Bridge\ReflectionReactDelegateFactory.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Reflection\EnumHelpers.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Reflection\ExpressionExtensions.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Reflection\ReflectionHelpers.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UIManager\AppRegistry.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UIManager\Events\Event.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UIManager\Events\RCTEventEmitter.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UIManager\Events\TouchEventType.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UIManager\Events\TouchEventTypeExtensions.cs" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
Rozele is probably going to complain about this but I can't find the option in Atom to disable it.